api/manifest/manifest_test.go

```diff
@@ -177,6 +177,37 @@ worker:
 	_assert(t, cases)
 }
 
+func TestGenerateDockerfileProcfile(t *testing.T) {
+	destDir := mkBuildDir(t, "../../examples/dockerfile-procfile")
+	defer os.RemoveAll(destDir)
+
+	Init(destDir)
+	m, _ := Read(destDir, defaultManifestFile)
+
+	Execer = func(bin string, args ...string) *exec.Cmd {
+		return exec.Command("true")
+	}
+
+	cases := Cases{
```

Comment:
@ddollar  The example in the test is a sinatra app with a Procfile. Do you want to see more examples? Or a different language/framework?